packages/rustcat/PKGBUILD

```diff
@@ -3,34 +3,35 @@
 
 pkgname=rustcat
 _binname=rc
-pkgver=129.15e116a
+pkgver=1.1.1
 pkgrel=1
 groups=("blackarch", "blackarch-networking")
-pkgdesc="Rustcat - Modern Port Listener & Reverse Shell"
+pkgdesc="Rustcat - A Modern Port Listener & Reverse Shell"
 url="https://github.com/robiot/rustcat"
 license=('MIT')
 arch=('x86_64')
 makedepends=(
     'rustup'
     'git'
 )
-md5sums=('SKIP')
+sha512sums=('SKIP')
 
 source=("$pkgname::git+https://github.com/robiot/rustcat.git")
 
 pkgver() {
   cd $pkgname
 
-  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+  echo $(git describe --tags --abbrev=0 | cut -c 2-)
```

Comment:
Use `git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'`, please follow https://github.com/BlackArch/blackarch-pkgbuilds

Reply:
Fixed it!